docs/rules/no-restricted-imports.md

```diff
@@ -0,0 +1,51 @@
+# Disallow Node imports (no-restricted-imports)
+
+Disallowing usage of specific node imports can be useful if you want to control the available methods, a developer can
```

Comment:
We probably want this to be a bit more generic in description. You can block any imports, not just those found in Node.js. So I'd change the description to "Disallow specific imports" and rework this introduction. 

This section should also have some background about the concept of importing and why you may want to restrict access to certain imports.